Resources/Prototypes/_Goobstation/Reagents/toxins.yml

```
@@ -184,6 +184,49 @@
+  name: reagent-name-mime-smoke
+  desc: reagent-desc-mime-smoke
+  color: "#0b8019"
+  physicalDesc: reagent-physical-desc-thick
+  metabolisms:
+    Poison:
+      effects:
+      - !type:Emote
+        conditions:
+        - !type:HasComponentCondition
+          components:
+            - MimePowers
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)